### ISSUE TITLE
layer freezing?

### ISSUE THREAD

**Issue by fdarvas** [roboflow/rf-detr#118]
For fine tuning, are there options to freeze layers? If not , would be a nice feature to have.

**Comment by isaacrob-roboflow** [comment]
We use an exponentially decaying learning rate on different parts of the model so layer freezing shouldn't be necessary. Why are you interested in it?

**Comment by fdarvas** [comment]
For fine-tuning the model. But it looks like that is already baked into the current training scheme via the decaying learning rate?

**Comment by isaacrob-roboflow** [comment]
Yep! This model supports fine tuning as is :)

**Comment by isaacrob-roboflow** [comment]
marking this as closed, feel free to reopen :)
